client/src/components/forms/select/autocomplete/component.tsx

```diff
@@ -109,6 +113,9 @@ const AutoCompleteSelect = <T,>({
               })}
             >
               {filteredOptions[index].label}
+              {filteredOptions[index].extraInfo && (
+                <span className="text-sm">{` - ${filteredOptions[index].extraInfo}`}</span>
+              )}
```

Comment:
I don't think this solution scales properly, we are assuming here a way to pass extra info to the value but given a certain format (` - {whatever}`), but what if I want to add info with a different format? I just can't.

Anyway, why don't just modify the label of the options?

Reply:
I just used the same property being used before (extraInfo), but I like your solution better. I will change it.